geode-core/src/distributedTest/java/org/apache/geode/internal/cache/persistence/PersistentPartitionedRegionWithRedundancyDUnitTest.java

```diff
@@ -143,14 +140,15 @@ public void recoversFromDisk() throws Exception {
    */
   @Test
   public void testGetDataDelayDueToRecoveryAfterServerShutdown() throws Exception {
-    int numBuckets = 10000;
+    int numEntries = 10000;
+    System.setProperty("gemfire.disk.recoverValuesSync", "true");
```

Comment:
I'm not sure this is required — It seems to pass on my machine with and without this line. Setting system properties can be problematic because it will affect how other tests run. If you do absolutely need to set a system property here, then you should find a way to set it back to the original value after the test has completed.

Reply:
Removed setting of system property, since it is no more needed (modified TC).